grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -155,12 +155,24 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
         };
       });
 
+  /** A variable to check if selectedClient$ has subscribers */
+  clientSubscribed: Boolean = false;
+
+  /** Stops polling updates for selectedClient$ */
+  deselectClient() {
```

Comment:
We try to have as little moving parts as possible. `clientSubscribed : Boolean` is explicit state that needs to be tested and can diverge from the actual fact if there is a subscriber to `selectedClient$` or not.

Please look at [`flowListEntries$`](https://github.com/google/grr/blob/master/grr/server/grr_response_server/gui/ui/store/client_page_facade.ts#L203) for how the polling is started and stopped only based on the subscription to an observable. The RxJS docs on [Subscribing to Observables](https://rxjs-dev.firebaseapp.com/guide/observable#subscribing-to-observables) and [Executing Observables](https://rxjs-dev.firebaseapp.com/guide/observable#executing-observables) might help.